frontend/styles/shared-styles.js

```diff
@@ -1,20 +1,49 @@
 import "@vaadin/vaadin-charts/theme/vaadin-chart-default-theme";
+import "@vaadin/vaadin-lumo-styles/all-imports";
 
 const $_documentContainer = document.createElement('template');
 
 $_documentContainer.innerHTML = `
+<dom-module id="common-styles">
+  <template>
+    <style include="lumo-color lumo-typography">
+        kbd {
+            font-family: var(--lumo-font-family);
+            display: inline-block;
+            background: var(--lumo-contrast-10pct);
+            padding: var(--lumo-space-wide-xs);
+            margin: 0 var(--lumo-space-wide-xs);
+            border-radius: var(--lumo-border-radius-s);
+            box-shadow: 0 0 0 1px var(--lumo-contrast-10pct), var(--lumo-box-shadow-xs);
+            font-size: var(--lumo-font-size-m);
+        }
+    </style>
+  </template>
+</dom-module>
+
 <custom-style>
-  <style>
+  <style include="common-styles">
+    html,
+    body {
+        height: 100%;
+    }
+  
     vaadin-app-layout vaadin-tab a:hover {
       text-decoration: none;
     }
   </style>
 </custom-style>
 
+<dom-module id="menu-bar-theme" theme-for="vaadin-menu-bar">
+  <template>
+    <style include="common-styles"></style>
```

Comment:
Actually I would prefer removing MenuBar from the Pricing example completely and just using plain inline links and Button (so they would just wrap normally on small viewport). But since I originally added the menu bar just to test it Sami wanted to keep it there though I'm not sure if this is a good way to use the MenuBar or not.

Basically `vaadin-menu-bar` has no examples about using links `<a>` inside it and only talks about buttons so I'm not sure should it have those (and include Lumo styles for `<a>` or not).

Reply:
Thanks. We can discuss what to do with the links later, if the users ask for it.